src/state/quick-bar-mixin.ts

```diff
@@ -28,4 +30,8 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
         }
       });
     }
+
+    private isOSCtrlKey(e: KeyboardEvent) {
+      return isMacOS ? e.metaKey : e.ctrlKey;
```

Comment:
Lets just add this to the if?

Reply:
I'm personally a fan of using tiny, well-named functions that explain, without code comments, why something's being done.

But I'm fine with whatever here.